src/Limelight.h

```diff
@@ -277,6 +277,10 @@ typedef struct _DECODE_UNIT {
 // number of slices per frame. This capability is only valid on video renderers.
 #define CAPABILITY_SLICES_PER_FRAME(x) (((unsigned char)(x)) << 24)
 
+// If set in the video renderer capabilities field, this flag specifies that the renderer
+// needs to opt-in to intra refresh, some clients (like the Xbox client with NVEnc) needs it
+#define CAPABILITY_INTRA_REFRESH 0x80
```

Comment:
Sorry for another nit, can you move this above the special `CAPABILITY_SLICES_PER_FRAME` macro too?